src/main/java/org/quartzpowered/common/factory/FactoryRegistry.java

```diff
@@ -33,7 +33,8 @@
 
 @Singleton
 public class FactoryRegistry {
-    @Inject private Injector injector;
+    @Inject
+    private Injector injector;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.